### PR TITLE
Feature/54 extend patient profile

### PR DESCRIPTION
**Pull request by serhiidotsyk**

### CODE REVIEW

File: src/app/Components/PatientProfile/update-patient/update-patient.component.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #GlobalDiv {
-  background-color: #e1e8f0;
+  // background-color: #ffffff;
```
Comment (TREN-D) [inline src/app/Components/PatientProfile/update-patient/update-patient.component.scss]: remove comment

File: src/app/Models/Illness-history.ts [hunk 2]
```diff
@@ -1,7 +1,13 @@
 export class IllnessHistory {
-    dateTime: any;
-    doctorName: string;
-    symptoms: string;
-    diagnosis: string;
-    treatment: string;
+    appointmentId: number;
+    conslusionTime: Date;
+    // doctorName: string;
```
Comment (TREN-D) [inline src/app/Models/Illness-history.ts]: here too

Comment (serhiidotsyk) [reply src/app/Models/Illness-history.ts]: done

File: src/app/Components/PatientProfile/update-patient/update-patient.component.scss (hunk 1 above)
Comment (serhiidotsyk) [reply src/app/Components/PatientProfile/update-patient/update-patient.component.scss]: done

**Review by martyniukroman — APPROVED** [review]

**Review by SergiyKozachok — APPROVED** [review]

**Review by TREN-D — APPROVED** [review]

**Review by edward0802 — APPROVED** [review]

**Pull request merged**